packages/core/src/sso/OidcConnector/utils.ts

```diff
@@ -20,30 +20,28 @@ import {
   type OidcTokenResponse,
 } from '../types/oidc.js';
 
+export const fetchOidcConfigRaw = async (issuer: string) => {
```

Comment:
ditto: Lack of providing sufficient context here.  What is the relation between our new SAML IdP implementation and this OidcConnector? 
 If we intend to use it to handle the OIDC requests, one good way is to create an OidcConnector in the SAML callback flow instead of importing random methods.

Reply:
Ditto, refactor this later in LOG-10600